nanome/_internal/_shapes/_serialization/__init__.py

```diff
@@ -2,4 +2,5 @@
 from ._sphere_serializer import _SphereSerializer
 from ._line_serializer import _LineSerializer
 from ._label_serializer import _LabelSerializer
+from ._mesh_serializer import _MeshSerializer
 from ._shape_serializer import _ShapeSerializer
```

Comment:
```suggestion
from ._shape_serializer import _ShapeSerializer

```

Reply:
Is the space mandatory ? Did I miss something else here ?

Reply:
Its a very minor python styling thing. You'll notice the warning "No newline at the end of file" in the diff.